### DISCUSSION TITLE
CUDA_ERROR_OUT _OF_MEMORY: out of memory

### DISCUSSION THREAD

**Discussion by WU2JING** [deepmodeling/deepmd-kit#568] [General]
When I try to make a simulation using lammps, however, I always meet the problem.  "CUDA_ERROR_OUT _OF_MEMORY: out of memory". I hope that you can help me to fix the problem. 
Any discussion is welcomed.Thank you very much.

**Comment by njzjz** [comment]
As is said, your GPU memory is not enough. You may reduce your system size or call MPI to run on multiple GPUs.

**Reply by njzjz** [reply]
Or change to another GPU card that has larger memory.
